include/can/core/messages.hpp

```diff
@@ -128,13 +127,13 @@ struct GetStatusResponse : BaseMessage<MessageId::get_status_response> {
 };
 
 struct MoveRequest : BaseMessage<MessageId::move_request> {
-    ticks duration;
+    can_ticks duration;
     um_per_tick velocity;
     um_per_tick_sq acceleration;
 
     template <bit_utils::ByteIterator Input, typename Limit>
     static auto parse(Input body, Limit limit) -> MoveRequest {
-        ticks duration = 0;
+        can_ticks duration = 0;
```

Comment:
Why the `can_ticks` name?

Reply:
Because firmware is receiving the duration in 32 bits but sending out 64 bits remaining duration, both are in ticks.